source/application/AMAppController.cpp

```diff
@@ -139,12 +139,28 @@ bool AMAppController::startup(){
 
 void AMAppController::shutdown()
 {
-	AMDatamanAppControllerForActions3::shutdown();
+	AMActionRunner3::releaseScanActionRunner();
+	AMActionRunner3::releaseWorkflow();
+
 	AMBeamline::releaseBl();
 	AMStorageRing::releaseStorageRing();
 	AMProcessVariableSupport::shutdownChannelAccess();
 }
 
+bool AMAppController::startupBeforeUserInterface()
```

Comment:
No. This is newly added codes. Try to initialize the workflow action runner and the scan action runner before we initialize UI.

This is also one of the main changes for this issue: synchronize the scanAction status between the workflow action runner and the scan action runner.